chronicle/Views/EnrolledView.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  EnrolledView.swift
+//  EnrolledView
+//
+//  Created by Giulia Campana on 8/12/21.
+//
+
+import SwiftUI
+
+struct EnrolledView: View {
+    var enrollmentViewModel: EnrollmentViewModel
+    let defaults = UserDefaults.standard
+    
+    var body: some View {
+        ScrollView {
+            VStack(alignment: .leading) {
+                AppHeader()
+                Text("Participant ID:").padding(.bottom)
+                Text(defaults.string(forKey: UserSettingsKeys.participantId) ?? "").padding(.bottom)
+                Text("Study ID:").padding(.bottom)
+                Text(defaults.string(forKey: UserSettingsKeys.studyId) ?? "").padding(.bottom)
+                Text("Organization ID:").padding(.bottom)
+                Text(defaults.string(forKey: UserSettingsKeys.organizationId) ?? "").padding(.bottom)
```

Comment:
No need to use UserDefaults here. You can access the organizationId, participantId and studyId directly from the viewModel object